benchmark/child_process/child-process-read.js

```diff
@@ -19,7 +19,7 @@ function main(conf) {
   const dur = +conf.dur;
   const len = +conf.len;
 
-  const msg = '"' + Array(len).join('.') + '"';
+  const msg = `"${'.'.repeat(len)}"`;
```

Comment:
I've run only the linter because it seems CI does not run benchmarks. There are some new tests for this, but they cover only `net` and `http` benchmarks for now. But I've run these files locally and have not found any difference or errors. However, to be on the safe side: @nodejs/benchmarking , @nodejs/performance — what do you think?